frontend/src/features/pages/LoggedOutPage.tsx

```diff
@@ -41,7 +41,7 @@ const LoggedOutPage = () => {
       ) : (
         <div style={{ textAlign: "center" }}>
           <Title order={2}>
-            Nope, you're still logged in,{" "}
+            {"Nope, you're still logged in, "}
```

Comment:
Why are the braces necessary here? I'm mostly curious as I wasn't able to find one by searching (it is a bit difficult to search :) )

Reply:
We leverage JSX here: we use an evaluated string literal, so we don't need to escape spaces and special characters.